### PR TITLE
Fixed Storybook

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

We'll be using Storybook for the design system, so I've resuscitated it. We're currently using a downgraded version (3.4.11) because we are currently using webpacker 3. Latest versions of Storybook have been on webpack 4 for quite some time now.

There is a bigger task of upgrading webpacker, so for now we will stay with this downgraded version of Storybook. See this [Tweet thread](https://twitter.com/nickytonline/status/1220356601066008581).

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Nick Miller from New Girl saying he's a great story teller

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,10 +57,10 @@
   },
   "homepage": "https://github.com/thepracticaldev/dev.to#readme",
   "devDependencies": {
-    "@storybook/addon-actions": "^5.1.9",
-    "@storybook/addon-links": "^5.1.9",
-    "@storybook/addons": "^5.1.8",
-    "@storybook/react": "^3.4.11",
+    "@storybook/addon-actions": "3.4.11",
```
Comment (nickytonline) [inline package.json]: This PR looks like nothing... it is. However, I was scratching my head for a whole day why downgrading didn't work. I finally literally copied the entire package.json version from the last working Storybook in the repo and it still wasn't working. And then it clicked. I've had this issue before. I had to delete the yarn.lock file and completely regenerate it, to have the  proper deps in the lock file. 🙃

**Comment by nickytonline** [comment]
@maestromac @Zhao-Andy, no rush on this, but this means at some point we can revisit #338.

**Comment by nickytonline** [comment]
Although I have not changed any of the code for the application, the yarn.lock file had some sever modifications. I would assume all is good, given the build status, but I will do a smoke test in the AM.

**Comment by nickytonline** [comment]
I've done a smoke test and things appear to be good in the app.

File: package.json (hunk 1 above)
Comment (maestromac) [reply package.json]: what a doozy

**Pull request merged**